autotask/pricers-bot/codebuild/base-asset/index.js

```diff
@@ -0,0 +1,138 @@

```

Comment:
same with this, can this be imported? :)

Reply:
Yes same as above, but those are the only two files that this happens.